_pages/governance.md

```diff
@@ -89,35 +89,37 @@ In addition to their actions as users, contributors may also find themselves doi
+New maintainers can be nominated by any existing maintainer. Once they have been nominated, there will be a vote by the project management committee (PMC; see below). Maintainer nomination and voting is one of the few activities that takes place on the project's private management list. This is to allow PMC members to freely express their opinions about a nominee without causing embarrassment. Once the vote has been held, the nominee is notified of the result. The nominee is entitled to request an explanation of any ‘no' votes against them, regardless of the outcome of the vote. This explanation will be provided by the [PMC Chair](#pmc-chair) and will be anonymous and constructive in nature.
 
-Nominees may decline their appointment as a committer. However, this is unusual, as the project does not expect any specific time or resource commitment from its community members. The intention behind the role of committer is to allow people to contribute to the project more easily, not to tie them into the project in any formal way.
+Nominees may decline their appointment as a maintainer. However, this is unusual, as the project does not expect any specific time or resource commitment from its community members. The intention behind the role of maintainer is to allow people to contribute to the project more easily, not to tie them into the project in any formal way.
 
-It is important to recognise that commitership is a privilege, not a right. That privilege must be earned and once earned it can be removed by the [PMC](#project-management-committee) in extreme circumstances. However, under normal circumstances committership exists for as long as the committer wishes to continue engaging with the project.
+It is important to recognise that maintainership is a privilege, not a right. That privilege must be earned and once earned it can be removed by the [PMC](#project-management-committee) in extreme circumstances. However, under normal circumstances Maintainership exists for as long as the maintainer continues engaging with the project.
 
-A committer who shows an above-average level of contribution to the project, particularly with respect to its strategic direction and long-term health, may be nominated to become a member of the PMC. This role is described below.
+A maintainer who shows an above-average level of contribution to the project, particularly with respect to its strategic direction and long-term health, may be nominated to become a member of the PMC. This role is described below.
+
+Note: The definitive list of maintainers is kept via a GitHub Team and lists on the website, etc. may be out of date.
```

Comment:
This note is somewhat word salad and I'm not sure how to parse it.  Do we need to say where the GitHub Team is kept?

Reply:
GitHub doesn't have an option to show memberships of teams unless you are part of the organization. So, there's no good way to automatically make sure everything is visible publicly. That's what this was trying to say. I've updated it to be clearer.

Reply:
Sure ... however what does "kept via a GitHub Team" mean then?

Reply:
![image](https://github.com/user-attachments/assets/87441c52-57b5-4d28-b470-3daa7ade6972)

![image](https://github.com/user-attachments/assets/51ddce48-a4e5-4554-bb69-b4c5893d61c6)

https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams

Could you give me a suggestion on how to update the sentence to make it more clear?

Reply:
> GitHub doesn't have an option to show memberships of teams unless you are part of the organization. So, there's no good way to automatically make sure everything is visible publicly. That's what this was trying to say. I've updated it to be clearer.

If the Teams page is not accessible to everyone we shouldn't be using it as main reference IMHO.
We already have a maintainers file within the gem5 repo: https://github.com/gem5/gem5/blob/stable/MAINTAINERS.yaml.

Can't we add a similar file for the PMC?

Reply:
I think @giactra 's suggestion is the way I would go about it, but if it's not possible then I think the current wording is ok.

Reply:
If @giactra is ok with it, it's fine with me -- I will wait to approve until he confirms.

Reply:
I was waiting on @giactra to ok before approving ... I'll give him a day to chime in?

Reply:
I guess I'm going to ok, @giactra let us know if this is a problem